### DISCUSSION TITLE
reset show/hide state to default

### DISCUSSION THREAD

**Discussion by bluelovers** [KevinVandy/material-react-table#392] [Ideas]
add a `reset` button

![image](https://user-images.githubusercontent.com/167966/220681907-41dba475-1236-4d66-9c30-ad9dd99cdb87.png)

**Comment by koxudaxi** [comment]
This has been fixed - duplicate types in Union are now correctly deduplicated (e.g., `Sequence[Union[float, float]]` becomes `Sequence[float]`).
